app/controllers/katello/api/v2/subscriptions_controller.rb

```diff
@@ -242,58 +255,29 @@ def index_organization
     end
 
     api :GET, "/systems/:system_id/subscriptions/available", N_("List available subscriptions"), :deprecated => true
-    param :system_id, String, :desc => N_("UUID of the system"), :required => true
-    param :match_system, :bool, :desc => N_("Return subscriptions that match system")
-    param :match_installed, :bool, :desc => N_("Return subscriptions that match installed")
-    param :no_overlap, :bool, :desc => N_("Return subscriptions that don't overlap")
-    def available_system
+    param :system_id, String, :desc => N_("UUID of the content host"), :required => true
+    param :match_system, :bool, :desc => N_("Return subscriptions that match a content host")
+    param :match_installed, :bool, :desc => N_("Return subscriptions that match installed products")
+    param :no_overlap, :bool, :desc => N_("Return subscriptions which do not overlap with a currently-attached subscription")
+    def available_for_system
```

Comment:
Was `available_for_system` the only one of the `available_for...` functions that was originally an API action?